person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = generate_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
+  end
+
```

Comment:
- [ ] [OPTIONAL] Please make the `of_age?` method private. The reason for this suggestion is that the `of_age?` method is only used internally by the `can_use_services?` method, and it is not meant to be called by other objects or classes. Making it private would prevent accidental or intentional misuse of the method, and also make the public interface of the `Person` class simpler and clearer. Additionally, making it private would allow the implementation of the method to change without affecting the external behavior of the class.

Reply:
thanks